### PR TITLE
added a NoticeAssignment to a passing time

### PR DESCRIPTION
**Pull request by ue71603**
addresses #734

### CODE REVIEW

File: examples/standards/epiap/EPIAP_Accessibility_Chur_V08_vehicletypes.xml [hunk 1]
```diff
@@ -2460,6 +2460,17 @@
...
 					</typesOfService>
+					<notices>
+						<Notice id="notice1" version="any">
+							<Text>Test notice on a PASSINGTIME</Text>
+						</Notice>
+					</notices>
+					<noticeAssignments>
+						<NoticeAssignment id="notice1" version="any" order="1">
+							<NoticeRef ref="notice1" version="any"/>
+							<NoticedObjectRef ref="ch:1:StopPointInJourneyPattern:91YYY_.j21-1-0020101:" version="any"/>
```
Comment (trurlurl) [inline examples/standards/epiap/EPIAP_Accessibility_Chur_V08_vehicletypes.xml:2471]:
```suggestion
							<NoticedObjectRef ref="ch:1:TimetabledPassingTime:10:91YYY_.j21-1-0020101" version="any"/>
```

Comment (trurlurl) [reply examples/standards/epiap/EPIAP_Accessibility_Chur_V08_vehicletypes.xml:2471]: That way I would understand it...

Comment (skinkie) [reply examples/standards/epiap/EPIAP_Accessibility_Chur_V08_vehicletypes.xml:2471]: Agreed.

**Comment by ue71603** [comment]
@skinkie @trurlurl @Aurige  ready for rereview

**Review by skinkie — APPROVED** [review]

**Review by trurlurl — APPROVED** [review]

**Pull request merged**